app/src/pages/inside/logsPage/logItemInfo/infoTabs/infoTabs.scss

```diff
@@ -47,6 +47,9 @@
     color: $COLOR--charcoal-grey;
     fill: $COLOR--charcoal-grey;
   }
+  &:nth-child(2):hover svg {
```

Comment:
It is unreliable to use selector by item order here.
It would be better to control this case by providing property f.e. `stroked` from tab config.